lib/consumer.ex

```diff
@@ -435,55 +452,26 @@ defmodule GenRMQ.Consumer do
 
   defp setup_consumer(%{in: chan, config: config, module: module} = state) do
     queue_config = config[:queue]
-    queue = QueueConfiguration.name(queue_config)
-    exchange = config[:exchange]
-    routing_key = config[:routing_key]
     prefetch_count = String.to_integer(config[:prefetch_count])
 
-    deadletter_args = setup_deadletter(chan, config)
-
-    arguments = QueueConfiguration.build_queue_arguments(queue_config, deadletter_args)
+    if queue_config.dead_letter[:create] do
```

Comment:
what is `dead_letter[:create]`? I might have missed it in description or sample config?

Reply:
It's basically the deadletter keyword on init. It's set in queue_configuration.ex on line 17.